docs/Local/installMPS.md

```diff
@@ -47,15 +47,17 @@ Find details about architectural details, security issues, and more in [Microser
 
     === "Linux"
         ``` bash
+        cd open-amt-cloud-toolkit
```

Comment:
This might be easiest if we just append it to the Step 2 cmd on Line 43. People seemed to have issues remembering to press Enter the second time in DXA2 if there were two cmds in one code block.  Example: 
``` bash
git clone --recursive --branch v1.1.0 https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit && cd open-amt-cloud-toolkit
```

Reply:
Done.